src/DIRAC/Resources/Storage/OccupancyPlugins/WLCGAccountingJson.py

```diff
@@ -43,7 +45,8 @@ def _downloadJsonFile(self, occupancyLFN, filePath):
                 if not res["OK"]:
                     continue
                 occupancyURL = res["Value"]
-                ctx.filecopy(params, occupancyURL, "file://" + filePath)
+                with setGfalSetting(ctx, "HTTP PLUGIN", "OPERATION_TIMEOUT", 30):
+                    ctx.filecopy(params, occupancyURL, "file://" + filePath)
```

Comment:
You also need to set the `params.timeout` value, and add a timeout in `requests` in `WLCGAccountHTTPJson` class.
Otherwise, looks good to me, thanks

Reply:
Done.